odl/tomo/geometry/geometry.py

```diff
@@ -154,6 +155,18 @@ def ndim(self):
         return self._ndim
 
     @property
+    def implementations(self):
```

Comment:
I was thinking something complicated, so users dont go for it. How about `implementation_cache`?